client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/notices/notices.tsx

```diff
@@ -0,0 +1,38 @@
+import { SnackbarList, withNotices } from '@wordpress/components';
```

Comment:
There are new functions and existing functions that are modified in this pull request. Can some tests be written for those functions to ensure correctness as well as communicate to others developers through the tests what expected results look like?

Reply:
Thanks for your comment. I'll consider adding tests on follow-up PRs.